docs/examples/json-mode.md

```
@@ -0,0 +1,134 @@
+<!-- Generated by scripts/docs/build-docs-data.ts. Do not edit by hand. -->
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)